pkg/controller/migmigration/backup.go

```diff
@@ -358,6 +443,93 @@ func (t *Task) annotateStorageResources() error {
 		}
 	}
 
+	return nil, resticPodCount
+}
+
+func (t *Task) areStagePodsCreated() (bool, error) {
```

Comment:
Just realized, don't we create these "stage pods" even when we're running a "final migration"? I suppose it makes sense if you think of a "stage" as the part of the process that something that always moves (stages) data over.